package/src/bin/thing-flinger.rs

```diff
@@ -0,0 +1,590 @@
+use omicron_package::{parse, SubCommand as PackageSubCommand};
```

Comment:
Should probably include the same license text here as other files.

Reply:
fixed